arbitrum/apibackend.go

```diff
@@ -130,8 +136,20 @@ func (a *APIBackend) GetArbitrumNode() interface{} {
 }
 
 // General Ethereum API
+func (a *APIBackend) SyncProgressMap() map[string]interface{} {
+	return a.sync.SyncProgressMap()
+}
+
 func (a *APIBackend) SyncProgress() ethereum.SyncProgress {
-	panic("not implemented") // TODO: Implement
+	progress := a.sync.SyncProgressMap()
```

Comment:
The impl looks good, but I'm curious, where is `SyncProcess()` called?

Reply:
from graphql's syncprogress - which requires the ethereum-defined struct